picard/formats/mp4.py

```diff
@@ -36,6 +38,22 @@
 from picard.util import encode_filename
 
 
+def _add_text_values_to_metadata(metadata, name, values):
+    for value in values:
+        value = value.decode("utf-8", "replace").strip("\x00")
+        metadata.add(name, value)
```

Comment:
IMHO, it could be written:

```python
metadata.add(name, value.decode("utf-8", "replace").strip("\x00")
```

without having too much readibility impact.